client/CalendarData/CalendarData.jsx

```diff
@@ -2,20 +2,97 @@
 const newDate = new Date();
 let currentDay = newDate.toDateString();
 currentDay = currentDay.replace(/2019/gi, '');
+const splitDay = currentDay.split(' ');
+currentDay = `${splitDay[0]}, ${splitDay[1]} ${splitDay[2]}`;
 
 // Grid and Calendar Data
 const availableTimes = ['5:00 PM', '5:30 PM', '6:00 PM', '6:30 PM', '7:00 PM', '7:30 PM', '8:00 PM', '8:30 PM', '9:00 PM'];
 
 const partySize = ['2 people', '3 people', '4 people', '5 people', '6 people', '7 people', '8 people'];
 
 const dayAbbreviations = ['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa'];
-const SeptemberDays = [];
+
+// Helper Functions to translate Date
+const getMonth = (inputDate) => {
+  if (inputDate === '8') {
+    return 'September';
+  } if (inputDate === '9') {
+    return 'October';
+  } if (inputDate === '10') {
+    return 'November';
+  }
+  return inputDate;
+};
+
+const translateMonth = (dateString) => {
+  const dateArr = dateString.split('-');
+  const num = +dateArr[1] - 1;
+  return getMonth(num.toString());
+};
+
+const translateDayOfWeek = (input) => {
+  switch (input) {
+    case 1:
+      return 'Mon';
+    case 2:
+      return 'Tue';
+    case 3:
+      return 'Wed';
+    case 4:
+      return 'Thu';
+    case 5:
+      return 'Fri';
+    case 6:
+      return 'Sat';
+    default:
+      return 'Sun';
+  }
+};
+
+const getString = (dateString) => {
+  if (dateString === '2019-10-01') {
+    return 'Tue, Oct 1';
+  }
+  if (dateString === '2019-11-01') {
+    return 'Fri, Nov 1';
+  }
+  const date = new Date(dateString);
+  const month = getMonth(date.getMonth().toString()).substring(0, 3);
+  let dayOfWeek;
+  let day;
+  if (month === 'Sep') {
+    dayOfWeek = translateDayOfWeek(date.getDay());
+    day = (date.getDate());
+  } else {
+    dayOfWeek = translateDayOfWeek(date.getDay() + 1);
+    day = (date.getDate() + 1);
```

Comment:
The declarations in this 'else' could be the initial values set on lines 61 and 62.
Non-blocking.